bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx128-eu-donate.html

```diff
@@ -0,0 +1,115 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_128_eu') }}
+{% endblock %}
+
+{% block experiments %}
+  {% if switch('experiment-firefox-whatsnew-128-eu') %}
+    {{ js_bundle('firefox_whatsnew_128_eu') }}
+  {% endif %}
+{% endblock %}
+
+{% if LANG == 'en-GB' %}
```

Comment:
I think English should probably go at the end as the `else` condition if all other conditions are false. Right now if you load the `en-US` locale with `?geo=GB` you get missing strings.

![image](https://github.com/mozilla/bedrock/assets/205591/2859c6e8-cd8c-47a5-af66-178e0afd7248)